Applications/Android/NNDetector/README.md

```diff
@@ -0,0 +1,47 @@
+# NNDetector
+NNDetector is a demo application conducting object detection based on nntrainer.
```

Comment:
Please add more explanation about this application, such as, which algorithm you use.. and what for.

Reply:
done